### PR TITLE
Add new plate reader compatibility. Update REVCOMP_SEQUENCERS

### PR DESCRIPTION
**Pull request by RodolfoSalido**

### CODE REVIEW

File: metapool/metapool.py [hunk 1]
```diff
@@ -66,17 +69,28 @@ def read_pico_csv(f, sep='\t', conc_col_name='Sample DNA Concentration'):
...
+                             skipfooter=15, engine='python')
+
+        pico_df = raw_df[['Wells','Concentration']]
+        pico_df = pico_df.rename(columns={'Concentration':conc_col_name,
+                                         'Wells':'Well'})
+
+        # coerce oddball concentrations to np.nan
+        pico_df[conc_col_name] = \
+        pd.to_numeric(pico_df[conc_col_name], errors = 'coerce')
+
```
Comment (ElDeveloper) [inline metapool/metapool.py:93]: How does this look to you @RodolfoSalido. Looking at the code I think we can take advantage of Python's default value for [`encoding`](https://docs.python.org/3/library/functions.html#open) in the `open` function. The other key thing is that `rename` will [ignore keys](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.rename.html) (see the errors argument) not in the table so we can enter all column names.

I also added a check to raise an exception if the plate reader is not a known value. Can you add a test to check that the exception is raised? For some examples search for `assertRaises` in the test_metapool.py file.

```suggestion
    if plate_reader == 'Synergy_HT':
        encoding, skipfooter = None, 5
    elif plate_reader == 'SpectraMax_i3x':
        encoding, skipfooter = 'utf-16', 15
    else:
        raise ValueError("Invalid plate reader %s" % plate_reader)

    pico_df = pd.read_csv(open(f, encoding=encoding), sep=sep, skiprows=2,
                          skipfooter=skipfooter, engine='python')

    # synergy's concentration column is "Concentration", spectramax's is
    # [Concentration]. Rename will ignore any labels not in the dataframe so
    # only one of the two label updates should happen
    pico_df.rename({'Concentration': conc_col_name,
                    '[Concentration]': conc_col_name,
                    'Wells': 'Well'}, inplace=True)

    pico_df = pico_df[['Well', conc_col_name]].copy()

    # coerce oddball concentrations to np.nan
    pico_df[conc_col_name] = \
        pd.to_numeric(pico_df[conc_col_name], errors='coerce')

    return pico_df
```

File: metapool/metapool.py [hunk 2]
```diff
@@ -48,7 +48,8 @@ def read_plate_map_csv(f, sep='\t'):
 
 
 # method to read minipico output
-def read_pico_csv(f, sep='\t', conc_col_name='Sample DNA Concentration'):
+def read_pico_csv(f, sep='\t', plate_reader='Synergy_HT',
+    conc_col_name='Sample DNA Concentration'):
```
Comment (ElDeveloper) [inline metapool/metapool.py:52]: This line needs to be aligned with the arguments in the previous line. If you run `flake8 metapool/metapool.py` you'll see the error I'm talking about.


This should fix it:
```suggestion
                  conc_col_name='Sample DNA Concentration'):
```

**Review by ElDeveloper — CHANGES_REQUESTED** [review]
Thanks so much @RodolfoSalido! Glad to see the list of sequencers getting an update. I just added a few minor comments, let me know if you want to chat about those offline or if something doesn't make sense.

:shipit:

File: metapool/metapool.py (hunk 2 above)
Comment (RodolfoSalido) [reply metapool/metapool.py:52]: sounds good, now I know how to check for pycodestyle.

File: metapool/metapool.py (hunk 1 above)
Comment (RodolfoSalido) [reply metapool/metapool.py:93]: looks good to me.

File: metapool/metapool.py [hunk 3]
```diff
@@ -66,17 +69,30 @@ def read_pico_csv(f, sep='\t', conc_col_name='Sample DNA Concentration'):
...
-    pico_df = raw_df[['Well', '[Concentration]']]
-    pico_df = pico_df.rename(columns={'[Concentration]': conc_col_name})
+    pico_df = pico_df[['Well', conc_col_name]].copy()
 
     # coerce oddball concentrations to np.nan
     pico_df[conc_col_name] = \
         pd.to_numeric(pico_df[conc_col_name], errors='coerce')
 
+    return pico_df
     return(pico_df)
```
Comment (ElDeveloper) [inline metapool/metapool.py:96]: My bad, I duplicated the last line:
```suggestion
```

**Review by ElDeveloper — APPROVED** [review]

**Comment by ElDeveloper** [comment]
@RodolfoSalido all looks good! Thanks so much. 🎆

**Pull request merged**